pkg/kvstore/logfields.go

```diff
@@ -47,4 +47,7 @@ const (
 
 	// fieldTTL is the TTL of a lease
 	fieldTTL = "ttl"
+
+	// fieldEndpoint is the etcd endpoint we talk to
+	fieldEndpoint = "etcdEndpoint"
```

Comment:
Maybe the variable name should be changed to `etcdEndpoint`?

Reply:
I used `fieldEtcdEndpoint` to keep with the style of the file. I'm not sure if that's what you had in mind.

Reply:
>I used fieldEtcdEndpoint to keep with the style of the file. I'm not sure if that's what you had in mind.

No, the issue is that you used `fieldEndpoint` not `fieldEtcdEndpoint` :-)